crates/runtime-table-partition/src/provider.rs

```diff
@@ -36,7 +36,7 @@ use crate::{
     insert::PartitionerExec,
 };
 
-mod pruning;
+pub mod pruning;
```

Comment:
```suggestion
pub(crate) mod pruning;
```

nit: This looks unused outside of this crate?

Reply:
Good catch, you're right. I opened it up to test part of it in the integration test and forgot to remove the `pub`. Thanks!